static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx

```diff
@@ -48,4 +50,8 @@ function Icon({type}: Props) {
   }
 }
 
+const IconHttp = styled(IconSort)`
```

Comment:
Should the icon just have a rotate prop?

Reply:
hmm good point! Thought of these rotated versions as edge cases (in the future we want to have a separate, better icon for `http`), so didn't think of adding a custom prop for them, but until then a prop is more neat. Added in https://github.com/getsentry/sentry/pull/32176/commits/e3d7cae9fac12b197b7fe1309d0e25443c3a57e8.